README.md

```diff
@@ -162,7 +162,12 @@ You can add an index to use approximate nearest neighbor search, which trades so
 Supported index types are:
 
 - [HNSW](#hnsw) - added in 0.5.0
+  - Better query performance than IVFFlat
```

Comment:
This isn't necessarily true. Based on how you construct and query your IVFFLAT index, pure query performance could be faster than HNSW. Where HNSW excels is that it typically has a better performance/recall ratio than IVFFLAT.

Reply:
Yeah, I feel like these bullet points would be better served as a dedicated "HMSW vs IVFFlat" section (or maybe in the FAQ).

Reply:
@jkatz - I pulled it from the README's existing text:

> It has better query performance than IVFFlat (in terms of speed-recall tradeoff), but has slower build times and uses more memory.